promitor-agent-scraper/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 2.1.0
+version: 2.1.1
```

Comment:
Can you revert this please? I'll do a release myself later on.

Reply:
Reverted.